action.yaml

```diff
@@ -53,7 +53,11 @@ runs:
         esac
 
         echo "Downloading UDS CLI ${version} for ${{ runner.os }} ${arch}"
-        curl -sL "${release_path}/uds-cli_${version}_${filename}" -o "${target_dir}/uds"
+        curl -f -sL "${release_path}/uds-cli_${version}_${filename}" -o "${target_dir}/uds"
+        if [ $? -ne 0 ]; then
```

Comment:
If this curl fails it exits the script and won't get to the next line to output this error message, see https://github.com/defenseunicorns/setup-uds/actions/runs/12380598836/job/34557339290?pr=12#step:3:59

Reply:
Updated in a5432bc